src/build/mod.rs

```diff
@@ -64,8 +64,13 @@ pub fn check_wasm_pack_versions() -> Result<WasmPackVersion, Error> {
     match wasm_pack_local_version() {
         Some(local) => {
             match Crate::return_wasm_pack_latest_version() {
-                Some(latest) => Ok(WasmPackVersion {local, latest}),
-                None => Ok(WasmPackVersion {local, latest: "".to_string()})
+                Ok(latest) => {
+                    match latest {
+                        Some(latest) => Ok(WasmPackVersion {local, latest}),
+                        None => Ok(WasmPackVersion {local, latest: "".to_string()})
+                    }
+                },
+                Err(err) => Err(err)
```

Comment:
This error propagation could be done slightly easier and more idiomatically with `?` and then providing a default for `None` is a bit nicer with `unwrap_or_else`. Put it together and we get:

```rust
Ok(Crate::return_wasm_pack_latest_version()?.unwrap_or_else(|| WasmPackVersion { ... }))
```

Reply:
Good suggestion, will do that. Thanks!